runtime-modules/argo-bridge/src/lib.rs

```diff
@@ -137,6 +137,33 @@ decl_module! {
             Ok(())
         }
 
+        // TODO: add weight for revert_outbound_transfer
+        #[weight = WeightInfoArgo::<T>::finalize_inbound_transfer()]
+        pub fn revert_outbound_transfer(
+            origin,
+            transfer_id: TransferId,
```

Comment:
I assume that this is an identifier whose validity is already confirmed by the squid.
Given that there are no checks on this...

Reply:
Namely the extr. assumes that `transfer_id` will refer to a pending transfer outbound request queued in the argo-squid

Reply:
Correct, likewise we don't check it any way when calling `finaliza_inbound_transfer`. It all assumes that the operator is acting correctly